### ISSUE TITLE
Battery Drain on Android 10

### ISSUE THREAD

**Issue by magnumforce2006** [daneren2005/Subsonic#976]
Since updating to android 10 I have noticed serious battery drain being caused by DSub. Selecting for android to "Optimize" the app seems to help, but this disables usable notification control.

**Comment by bjhiltbrand** [comment]
Agreed. I average 10 to 15 percent overall battery drain on my OnePlus 6 in between charges. This is without ever opening DSub after a phone reboot. I have finally resorted to deleting the app when not listening to music, and reinstalling it when I want to listen to music. This is a great player, and I would love to see this fixed as soon as possible.

**Comment by markmghali** [comment]
Yes I also have massive battery drain on Android 10. Is DSUB still in active development?

**Comment by pszabo75** [comment]
Absolutely not. If you check out the play store page of Dsub, the last update was in September 2018. Pull requests here never being merged. Unfortunately it seems to be a dead project.

**Comment by markmghali** [comment]
Well shoot I just paid for it recently.

**Comment by pbanj** [comment]
I'm on Android 10 and i haven't noticed any battery drain issues because of dsub. Hell i fell asleep while listening to it and my battery only went down 13% over the 9hrs i was sleeping and it was still playing when i woke up.

As for this being dead, if you check the commit history there were some 5 months ago.

**Comment by bjhiltbrand** [comment]
As a follow up, I noticed in this [open bug](https://github.com/daneren2005/Subsonic/issues/751) from 2016 that having casting on was the culprit back then. I toggled casting off and it fixed my battery drain issues. I toggled it back on and it started draining again. Someone over there mentioned that they tried the same thing and it didn't fix it for them. I don't know, it seems like it's still a problem somehow.
